public/app/plugins/panel/timeseries/plugins/AnnotationEditorPlugin.tsx

```diff
@@ -31,25 +31,21 @@ export const AnnotationEditorPlugin: React.FC<AnnotationEditorPluginProps> = ({
       plotInstance.setSelect({ top: 0, left: 0, width: 0, height: 0 });
     }
     setIsAddingAnnotation(false);
-  }, [setSelection, , setIsAddingAnnotation, plotCtx]);
+  }, [plotCtx]);
```

Comment:
I wonder how com eslint does not complain about missing dependencies in the hook list (i.e. setIsAddingAnnotation). Any ideas?

Reply:
i assume the lint plugin got smart enough to ignore hook setters, which i've always figured to be referentially consistent.